angular/package.json

```diff
@@ -7,21 +7,21 @@
     "build": "ng build"
   },
   "dependencies": {
-    "@angular/animations": "^20.0.0",
```

Comment:
This package is now deprecated, I'd remove it from the default project created on stackblitz